crates/rpc/rpc-types/src/eth/trace/parity.rs

```diff
@@ -394,17 +411,17 @@ impl Serialize for LocalizedTransactionTrace {
         s.serialize_field("subtraces", &self.trace.subtraces)?;
         s.serialize_field("traceAddress", &self.trace.trace_address)?;
 
-        match &self.trace.action {
-            Action::Call(_call_action) => {
+        match &self.trace.action.kind() {
```

Comment:
this can be simplified with a single `s.serialize_field("type", kind())?;` call if ActionKind serializes accordingly